lib/record_store/provider/dynect.rb

```diff
@@ -83,15 +87,21 @@ def secrets
     end
 
     def build_from_api(api_record)
+      record_type = api_record.fetch('record_type')
       record = api_record.merge(api_record.fetch('rdata')).slice!('rdata').symbolize_keys
 
-      return if record.fetch(:record_type) == 'SOA'
+      return if record_type == 'SOA'
+
+      case record_type
+      when 'ALIAS'
+        record[:cname] = record.delete(:alias)
+      end
```

Comment:
Breaking changes are fine in my opinion. We just need to increment a major version when cutting a new package.